modules/installation-infrastructure-bare-metal.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-infrastructure-bare-metal-{context}"]
+= Creating the bare metal infrastructure
+
+Before you deploy a {product-title} cluster on bare metal, you must
+create supporting infrastructure.
+
+.Procedure
```

Comment:
@abhinavdahiya, is there anything else that an admin needs to configure up front? Do we have any other requirements for these components?

Reply:
this seems correct.